src/Typechecker.mli

```diff
@@ -7,6 +7,14 @@ type error =
     (** var location, message *)
   | Env.error ]
 
+val makeUnboundRegularVarInsideBoxError :
+  Location.t ->
+  string ->
+  [> `UnboundRegularVarInsideBoxError of Location.t * string ]
+(** [makeUnboundRegularVarInsideBoxError loc var_name] create
+    [`UnboundRegularVarInsideBoxError] with human readable message.
+ *)
+
```

Comment:
Do we need to expose this function? Is there a use for it outside the typechecker module?

Reply:
Certainly no, fixed